interp/constrextern.ml

```diff
@@ -969,7 +977,8 @@ let rec extern inctx (custom,scopes as allscopes) vars r =
       CCast (sub_extern true scopes vars c,
              map_cast_type (extern_typ scopes vars) c')
   | GInt i ->
-     CPrim(Numeral (Uint63.to_string i,true))
+     extern_prim_token_delimiter_if_required
+       (Numeral (Uint63.to_string i,true)) "int63" "int63_scope" (snd scopes)
```

Comment:
Would it be possible to use `Register` to register a scope for printing primitive integers, rather than hard-coding this here?

Reply:
Hi @JasonGross, The `int63_scope` is indeed registered in `int63_syntax.ml`, but I'm unsure whether the code I proposed in this PR could be inserted in another place… Indeed IINM, this scope is a bit different than, e.g., real_scope, because the primitive integers pretty-printing can't be disabled by doing `Set Printing All` or so… Anyway, I'll be glad to refactor my PR if need be.

@maximedenes WDYT?

Reply:
@JasonGross that would indeed be nice but Register doesn't seem to support scopes currently. I may look into it further but that's definitely beyond the scope (no pun intended) of this PR.

Reply:
I agree it would be nice, but can be done later.